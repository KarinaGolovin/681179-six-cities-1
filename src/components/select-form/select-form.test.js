import React from 'react';
import renderer from 'react-test-renderer';
import SelectForm from './select-form.jsx';

it(`SelectForm correctly renders`, () => {
  const component = renderer.create(
      <SelectForm onSortTypeChange={() => {}}/>
  ).toJSON();

  expect(component).toMatchSnapshot();
});
