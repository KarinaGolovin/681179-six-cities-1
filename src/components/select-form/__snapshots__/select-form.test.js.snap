// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectForm correctly renders 1`] = `
<form
  action="#"
  className="places__sorting"
  method="get"
>
  <span
    className="places__sorting-caption"
  >
    Sort by
  </span>
   
  <span
    className="places__sorting-type"
    onClick={[Function]}
    tabIndex="0"
  >
    
    <svg
      className="places__sorting-arrow"
      height="4"
      width="7"
    >
      <use
        xlinkHref="#icon-arrow-select"
      />
    </svg>
  </span>
  <ul
    className="places__options places__options--custom "
  />
  <select
    className="places__sorting-type visually-hidden"
    id="places-sorting"
    onChange={[Function]}
  />
</form>
`;
